src/handlers.ts

```diff
@@ -35,21 +36,23 @@ export function getBuildImage(req: Request, res: Response, next: NextFunction):
         .catch(next);
 }
 
-export function enqueueBuild(req: Request, res: Response, next: NextFunction): void {
-    Promise.resolve()
-        .then(() => assertIdPathParam(req))
-        .catch(next);
+export async function enqueueBuild(req: Request, res: Response, next: NextFunction): Promise<void> {
+    const { originalname } = await req.file;
```

Comment:
is req.file really a promise?

if not you can just destructure it normallly with `const {originalname} = req.file`